scout/adapter/mongo/base.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, database=None):
     def init_app(self, app):
         """Setup via Flask."""
         dbname = app.config["MONGO_DBNAME"]
-        LOG.debug(f"Database name: {dbname}")
+        LOG.info(f"Database name: {dbname}")
```

Comment:
Make the test happy. I guess we defaulted to debug level up until just now, but does someone understand why and what changed? New version of pytest? Or click?

Reply:
No idea